docker-compose.dev.yml

```diff
@@ -0,0 +1,24 @@
+version: "2.3"
```

Comment:
I included this change so I could run `db` tests without having to forward and un-forward ports -- if you want to run the `db` tests, `source` a .env.local` in the terminal session where you're running sbt, `docker-compose -f docker-compose.dev.yml up` somewhere else, and then you can run `db/test ...` commands. Here's my `.env.local`:

```
# Development Settings
export POSTGRES_USER=rasterfoundry
export POSTGRES_PASSWORD=rasterfoundry
export POSTGRES_DB=rasterfoundry
export POSTGRES_URL=jdbc:postgresql://localhost:5432/
export FLYWAY_MIGRATIONS_DIR=filesystem:/home/james/gitdirs/raster-foundry/app-backend/db/src/main/resources

# Rollbar Settings
export ROLLBAR_SERVER_TOKEN=PLACEHOLDER
export ROLLBAR_CLIENT_TOKEN=PLACEHOLDER

# Memcached settings
export MEMCACHED_HOST=localhost
```